docs/docs/sample-scripts/spo/manage-news-page-types/index.md

```diff
@@ -0,0 +1,45 @@
+---
+tags:
+  - customizations
+  - libraries
+---
+
+# Create custom views to differentiate SharePoint news page types in Site Pages library
```

Comment:
Let's make this a little bit shorter, right now it's quite huge in the navigation.

```suggestion
# Create custom views to differentiate SharePoint news page types
```